src/ansys/templates/python/pyace_pckg/{{cookiecutter.__project_name_slug}}/docker-compose.yaml

```diff
@@ -0,0 +1,38 @@
+version: '3.5'
+
+services:
+
+  {{ cookiecutter.library_name }}:
```

Comment:
The `library_name` variable might include spaces or symbols. I am not familiar with `docker-compose.yml` file. Wouldn't it be better to use the `{{ cookiecutter.__library_name_slug }}`?

Reply:
Sure, that makes sense and I had changed after some test